plugins/dell-kestrel/fu-dell-kestrel-plugin.c

```diff
@@ -408,6 +408,36 @@ fu_dell_kestrel_plugin_modify_config(FuPlugin *plugin,
 	return fu_plugin_set_config_value(plugin, key, value, error);
 }
 
+static gboolean
+fu_dell_kestrel_plugin_backend_device_removed(FuPlugin *plugin, FuDevice *device, GError **error)
+{
+	const gchar *cache_keys[] = {"ec", "mst", "usb4", NULL};
```

Comment:
drop the NULL here, and then use `i < G_N_ELEMENTS(cache_keys)` in the loop pls.